Packs/Arcanna/ReleaseNotes/1_2_0.md

```diff
@@ -0,0 +1,24 @@
+#### Integrations
+
+##### Playbooks 
```

Comment:
Please use the correct format with update-release-notes command in the SDK. 
The playbooks shouldn't be under the intergations